turtl_backup.py

```diff
@@ -90,6 +90,12 @@ def build_basic_auth(auth_token):
     return 'Basic ' + b64encode(b'user:' + auth_token).decode()
 
 
+def hide_password(password):
```

Comment:
I really don't like this, can you please just not print the password at all ? Not even `***`, no even `"and password:"`, just no password?

Reply:
fix in 7a469f4dd16d82